build_tools/services.rb

```diff
@@ -123,6 +124,22 @@ def model_path(model_name, models_dir)
       File.exists?(path) ? path : nil
     end
 
+    def eventstream?(api)
+      api['shapes'].each do |_, ref|
+        return true if ref['eventstream'] || ref['event']
+      end
+      false
+    end
+
+    def core_version(api)
+      version_file = File.read("#{$GEMS_DIR}/aws-sdk-core/VERSION").rstrip
+      if eventstream?(api)
+        EVENTSTREAM_CORE_VERSION
```

Comment:
This is not right, we're going to lock customers on an old version of core. We need a bounded dependency like:

"~> 3", ">= 3.21.0"

Reference: https://guides.rubygems.org/patterns/#declaring-dependencies